src/modules/actions.js

```diff
@@ -38,6 +38,17 @@ export interface CurrencyEngineChangedTxs {
   };
 }
 
+/**
+ * Called when a currency engine fires the onTxidsChanged callback.
+ */
+export interface CurrencyEngineChangedTxids {
+  type: 'CURRENCY_ENGINE_CHANGED_TX_IDS';
```

Comment:
I'd like to see `TXIDS` as one word. Most Bitcoin folks treat `txid` as a single word.